entities/params/campaigns.go

```diff
@@ -24,3 +26,10 @@ type SendCampaign struct {
 func (p *SendCampaign) TrimSpaces() {
 	p.FromName = strings.TrimSpace(p.FromName)
 }
+
+type CampaignSchedule struct {
+	ScheduledAt string `form:"scheduled_at" validate:"required,max=191"`
```

Comment:
I think we can validate for datetime as well?

Reply:
@FilipNikolovski No but we can create custom validator for date time?

Reply:
The v10 validator has this I think: https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Datetime we should test it out.